File: MajorityElement/MajorityElement.java
public class Solution {
    public int majorityElement(int[] nums) {
        int majority=nums[0];
        int count=1;
        for(int index=1;index<nums.length;index++){
            if(count==0){
                count++;
                majority=nums[index];
            }else if(majority==nums[index]){
                count++;
            }else{
                count--;
            }
        }
        return majority;
    }
}

File: MajorityElement/majority_element.py
class Solution(object):
    def majorityElement(self,nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        major = nums[0]
        count = 1
        for index in range(1, len(nums)):
            if count == 0:
                count += 1
                major = nums[index]
            elif major == nums[index]:
                count += 1
            else:
                count -= 1
        return major